client/me/reauth-required/index.jsx

```diff
@@ -207,7 +207,8 @@ const ReauthRequired = createReactClass( {
 				<form onSubmit={ this.submitForm }>
 					<FormFieldset>
 						<FormLabel htmlFor="code">{ this.props.translate( 'Verification Code' ) }</FormLabel>
-						<FormTelInput
+						<FormTextInput
```

Comment:
In https://github.com/Automattic/wp-calypso/pull/23176/files#diff-b1f44d7b8ab7f6ed4e06bc2d9e1a5d23R332
`FormVerificationCodeInput` is used, shouldn't we use that?

Reply:
You're right, updating 👍